youths/admin.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib import admin
+from django.utils.translation import ugettext_lazy as _
+from reversion.admin import VersionAdmin
+
+from youths.models import YouthProfile
+
+
+@admin.register(YouthProfile)
+class YouthProfileAdmin(VersionAdmin):
+    list_display = (
+        "__str__",
+        "membership_number",
+    )
+    readonly_fields = (
+        "user",
+        "membership_number",
+        "approved_time",
+        "approval_notification_timestamp",
+    )
+    search_fields = (
+        "membership_number",
+        "user__first_name",
+        "user__last_name",
+        "school_name",
+        "school_class",
+    )
+    fieldsets = (
```

Comment:
You could add AdditionalContactPerson management here as well, since it's in the model.